backend/src/main/java/org/eclipse/tractusx/semantics/registry/controller/AssetAdministrationShellApiDelegate.java

```diff
@@ -118,21 +116,28 @@ public ResponseEntity<SubmodelDescriptor> getSubmodelDescriptorByIdThroughSuperp
 
     @Override
     public ResponseEntity<AssetAdministrationShellDescriptor> postAssetAdministrationShellDescriptor( AssetAdministrationShellDescriptor assetAdministrationShellDescriptor ) {
-        Shell saved = shellService.save(shellMapper.fromApiDto(assetAdministrationShellDescriptor));
+        Shell shell = shellMapper.fromApiDto(assetAdministrationShellDescriptor);
```

Comment:
It would be better to handle this settings in the service layer or much better in the mapper